vng_api_common/inspectors/view.py

```diff
@@ -393,4 +406,16 @@ def _get_response_headers_for_code(self, status_code, direction="response") -> d
             result[parameter.name] = parameter_type
         return result
 
+    def _map_serializer_field(self, field, direction, bypass_extensions=False):
+        meta = self._get_serializer_field_meta(field, direction)
+        if isinstance(field, HistoryURLField):
+            method = getattr(field.parent, field.method_name, None)
+            hint = self._map_response_type_hint(method)
+            hint["description"] = (
+                f"URL referenties van de {MAPPING_MODELS[field._kwargs.get('resource')]} welke horen bij deze versie van het ZAAKTYPE.",
+            )
+            return append_meta(hint, meta)
 
+        return super(AutoSchema, self)._map_serializer_field(
+            field, direction, bypass_extensions=False
```

Comment:
Willen we hier inderdaad `bypass_extensions` op `False` zetten? Dit wijkt af van het standaard gedrag maar misschien heb je hier bewust voor gekozen.

Reply:
in openapi.py staat de super op:

  def _map_serializer_field(self, field, direction, bypass_extensions=False):

of zie ik dit verkeerd?

Reply:
Dat zie je niet verkeerd, het standaard gedrag heb je hier inderdaad aangehouden. Ik heb verkeerd zitten kijken. Het lijkt mij beter om deze code in een `OpenApiSerializerFieldExtension` te stoppen, die zijn voor dit soort usecases bedoeld.